### PR TITLE
[PRM-1143] Removed LSP summary card link ID and updated header

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

**Comment by JBlacklaws** [comment]
Screenshot 2022-03-09 at 11 30 32
Screenshot 2022-03-09 at 11 30 53

File: app/views/IndexView.scala.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h1 class="govuk-heading-xl">@{messages("index.heading")}</h1>
             }
 
             @if(!user.isAgent){
-                <h3 class="govuk-heading-s">@getMessage("whatIsOwed.details.heading")</h3>
+                <h2 class="govuk-heading-s">@getMessage("whatIsOwed.details.heading")</h2>
```
Comment (LewisMitchell) [inline app/views/IndexView.scala.html:58]: Do we need to have the getMessage function call here?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
We still need to have the getMessage function in other places, I just meant in that particular use case

**Comment by LewisMitchell** [comment]
So just changing this:
`<h2 class="govuk-heading-s">@getMessage("whatIsOwed.details.heading")</h2>`
to:
`<h2 class="govuk-heading-s">@messages("whatIsOwed.details.heading")</h2>`

**Review by LewisMitchell — APPROVED** [review]
All good, just waiting on AT's

**Pull request merged**